extensions/amp-ad-network-fake-impl/0.1/amp-ad-network-fake-impl.js

```diff
@@ -59,15 +59,24 @@ export class AmpAdNetworkFakeImpl extends AmpA4A {
           // and the signature is "FAKESIG". This mode is only allowed in
           // `localDev` and primarily used for A4A Envelope for testing.
           // See DEVELOPING.md for more info.
-          const creative = this.transformCreativeLocalDev_(deserialized);
+          let creative = this.transformCreativeLocalDev_(deserialized);
```

Comment:
I checked at HEAD, and this does not currently seem to work. Unfortunately none of the team was aware of this URL format, so it's likely been broken for quite some time. If it's alright with you, I would suggest that we create a bug to fix that separately, since the current PR does not appear to be the one to have broken this functionality.